### PR TITLE
Don't use nonNegativeInteger as the type of the call-id attribute in API schema

### PR DESCRIPTION
**Pull request by clumens**

### CODE REVIEW

File: xml/Makefile.am [hunk 1]
```diff
@@ -162,6 +162,21 @@ api/api-result-%.rng: $(API_build_copies) best-match.sh Makefile.am
...
+	$(AM_V_at)echo '    <start>' >> $@
+	$(AM_V_at)echo '        <ref name="element-crm_mon-old"/>' >> $@
+	$(AM_V_at)echo '    </start>' >> $@
+	$(AM_V_at)echo '    <define name="element-crm_mon-old">' >> $@
+	$(AM_V_at)echo '        <element name="crm_mon">' >> $@
+	$(AM_V_at)echo '            <attribute name="version"> <text/> </attribute>' >> $@
+	$(AM_V_at)echo '            <externalRef href="$(<)" />' >> $@
+	$(AM_V_at)echo '        </element>' >> $@
+	$(AM_V_at)echo '    </define>' >> $@
+	$(AM_V_at)echo '</grammar>' >> $@
```
Comment (kgaillot) [inline xml/Makefile.am:178]: The final line should use AM_V_SCHEMA. This is a hack for files created by multiple lines.

This comes into play if someone does "./configure --enable-quiet" or "make V=0". Normally, make echoes every line it executes, but with one of those options, all of the AM_V_at lines will be silent and AM_V_SCHEMA will output "SCHEMA *filename*" to indicate that the file was generated.

**Review by kgaillot — COMMENTED** [review]
Looks good

File: xml/Makefile.am (hunk 1 above)
Comment (kgaillot) [reply xml/Makefile.am:178]: Actually maybe it's ./configure --enable-silent-rules. The abundance of "quiet" options is confusing ...

Comment (clumens) [reply xml/Makefile.am:178]: I got into a bit of a rhythm ther with the copy & paste at the front of each line.  Fixed and verified with "make V=0".

**Comment by kgaillot** [comment]
CI failure is from make distcheck:

    11:29:56 ERROR: files left in build directory after distclean:
    11:29:56 ./xml/crm_mon.rng

so it just needs to be added to some clean target

**Pull request merged**